packages/access-api/src/types/delegations.ts

```diff
@@ -0,0 +1,28 @@
+import * as Ucanto from '@ucanto/interface'
+
+export type Resolvable<T> = T | Promise<T>
```

Comment:
Do we need this right now? Is `Promise<T>` ok? Commonly in nodejs you'd find `*Sync` suffixed versions.

Reply:
We have `Await` type in multiformats for this and I think it’s also used and reexported by ucanto

Reply:
We don't *need* it. But right now an `Array` satisfies the `DelegationsStorage` type and getting rid of it would remove that property which, for sure, we don't need. (I started with an `Array` and expanded the type from there to afford for async and adding `Resolvable` was an easy way of doing that)

Reply:
Removed `Resolvable` in https://github.com/web3-storage/w3protocol/pull/427/commits/022dc2167647dcbabb35b6ab0450dafd9c708a84#diff-0c692d0f8036f645c9fe7c647d216265dedfb7a84decb52490918e43750a9e9eL3